### PR TITLE
added homescreen

### PR DESCRIPTION
**Pull request by muralimgwl**

### CODE REVIEW

File: web/ui-app-v2/src/components/AppBar/index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import AppBar from 'material-ui/AppBar';
+import PropTypes from "prop-types";
+
+const AppBarUi = (props) => {
+  return (
```
Comment (varun93) [inline web/ui-app-v2/src/components/AppBar/index.js]: Does this work?

Comment (muralimgwl) [reply web/ui-app-v2/src/components/AppBar/index.js]: yes

**Comment by varun93** [comment]
Doesn't work.

**Comment by varun93** [comment]
Merged Murali's feature.

**Pull request merged**